frappe/public/js/frappe/form/controls/select.js

```diff
@@ -117,8 +118,13 @@ frappe.ui.form.add_options = function(input, options_list) {
 			var is_label_null = is_null(v.label);
 			var is_disabled = Boolean(v.disabled);
 			var is_selected = Boolean(v.selected);
+			var is_group = false;
 
 			if (is_value_null && is_label_null) {
+			    if (v.startsWith('#')) {
```

Comment:
any existing options that had # in front won't be selectable after this right?

Reply:
Yes, but the code can be modified to handle that.